### PR TITLE
change lxt limits to +/-100us

### PR DESCRIPTION
**Pull request by vespos**
## Description
Update the lxt default limits to 100us.

## Motivation and Context

This is needed when laser timing is far back for some reason

## How Has This Been Tested?
Live when laser timing was bad.

## Where Has This Been Documented?
N/A

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- LaserTiming
+- LaserTimingCompensation
+- LxtTtcExample
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:16]:
```suggestion
- Updates limits for LaserTiming, LaserTimingCompensation, and LxtTtcExample from +/-10us to +/-100us
```

Maybe something like this?  I'd combine your comment into one line to make sure the change info is kept together

**Review by tangkong — COMMENTED** [review]
LGTM, aside from some pre-release note nitpicks as always

File: docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst (hunk 1 above)
Comment (ZLLentz) [reply docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:16]: I think the release notes here are the only adjustments needed to the PR imo

File: docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- Update limits from +/-10us to +/-100us
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:26]:
```suggestion
- N/A
```
Sorry I meant to imply this to go along with the previous suggestion

File: docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Contributors
+------------
+- espov
```
Comment (klauer) [inline docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:30]:
```suggestion
- vespos
```

Comment (vespos) [reply docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:30]: Do we use github or slac username here?

Comment (ZLLentz) [reply docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst:30]: I'm going to force accept this suggestion, approve, and merge after the ci re-runs

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**